### PR TITLE
Showing milestones

### PR DESCRIPTION
**Pull request by seejee**
This was actually pretty tricky. We have to parse out the linked issue number from our PRs and then cross reference that against the storyboard issues in order to find the proper milestone.

This PR also includes some UI cleanups to the issue list and renames "Dash" to "My Code Reviews".
## Note: Add a new ENV variable to your config

Add this to your secret_env and/or the Heroku app:

`DW_GH_STORYBOARD_REPO=storyboard`

### CODE REVIEW

**Comment by seejee** [comment]
@joelmccracken @srobbibaro Ready for review!

**Comment by srobbibaro** [comment]
Some failed tests.

File: lib/dev_wizard/github_gateway/issue.ex [hunk 1]
```diff
@@ -1,6 +1,8 @@
 defmodule DevWizard.GithubGateway.Issue do
   alias DevWizard.GithubGateway.{User, Comment, Milestone}
 
+  @linked_issue_regex ~r/(closes|connects)\s+thinkthroughmath\/storyboard#([0-9]+)/i
```
Comment (srobbibaro) [inline lib/dev_wizard/github_gateway/issue.ex]: You may want to expand this list a bit. I know there are other keywords Waffle supports and I definitely see people on our team use (at the minimum) "fixes."

Comment (srobbibaro) [inline lib/dev_wizard/github_gateway/issue.ex]: https://github.com/waffleio/waffle.io/wiki/FAQs#prs-closing-keywords

File: web/controllers/page_controller.ex [hunk 2]
```diff
@@ -12,10 +12,9 @@ defmodule DevWizard.PageController do
     |> render("index.html")
   end
 
-  def dash(conn, _params) do
+  def my_code_reviews(conn, _params) do
```
Comment (srobbibaro) [inline web/controllers/page_controller.ex:15]: 👍 👍 ++

File: web/templates/page/issue_list.html.eex [hunk 3]
```diff
@@ -56,10 +55,9 @@
                             <% end %>
                         </td>
                         <td>
-                            <%= pr.updated_at %>
-                        </td>
-                        <td>
-                            <%= pr.created_at %>
+                          Updated: <%= days_since(pr.updated_at) %>
+                          <br/>
```
Comment (srobbibaro) [inline web/templates/page/issue_list.html.eex]: 💔

File: web/views/page_view.ex [hunk 4]
```diff
@@ -1,11 +1,16 @@
 defmodule DevWizard.PageView do
   use DevWizard.Web, :view
+  use Timex
 
-  def review_state(issue) do
-    case issue.review_state do
-      :not_signed_off -> "Needs Review!"
-      :signed_off     -> "Signed Off"
-      _               -> "?"
+  def days_since(date) do
```
Comment (srobbibaro) [inline web/views/page_view.ex:5]: This helper is really nice. I like having it and I think displaying this information as "some amount of time ago" is the way to go. I'd like to think longer term if days and or hours is a more interesting measurement, but I think this is a great place to start. 👍

**Comment by srobbibaro** [comment]
Great stuff! Look into the broken tests and consider updating the waffle link regex to include additional closes/connects keywords and I'm GTG! 👍 🎆

File: web/views/page_view.ex (hunk 4 above)
Comment (seejee) [inline web/views/page_view.ex:5]: Good call! This is something we can definitely add later. And there's an opportunity here to open-source a bunch of rails-esque helpers for formatting Timex dates.

File: web/templates/page/issue_list.html.eex (hunk 3 above)
Comment (seejee) [inline web/templates/page/issue_list.html.eex]: Fixed!

File: lib/dev_wizard/github_gateway/issue.ex (hunk 1 above)
Comment (seejee) [inline lib/dev_wizard/github_gateway/issue.ex]: Good catch! Check the last commit. I added tests for all these cases too.

**Comment by seejee** [comment]
Tests are all fixed. Upgrading Timex changed the API for doing date math in the Cache module. Yay @joelmccracken for writing those tests!

File: lib/dev_wizard/github_gateway/cache.ex [hunk 5]
```diff
@@ -14,8 +14,9 @@ defmodule DevWizard.GithubGateway.Cache do
       Logger.debug "Cache  hit: #{inspect key}"
       results = Map.get(state, key)
 
-      earliest_to_keep = Date.now(:secs) - time_to_keep
-      if results[:timestamp] < earliest_to_keep do
+      diff = Timex.DateTime.diff(Timex.DateTime.now, results[:timestamp], :seconds)
+
+      if diff > time_to_keep do
```
Comment (seejee) [inline lib/dev_wizard/github_gateway/cache.ex:19]: @joelmccracken Here's the new (more sane) way of doing date math with Timex.

File: lib/dev_wizard/github_gateway/issue.ex [hunk 6]
```diff
@@ -1,7 +1,7 @@
 defmodule DevWizard.GithubGateway.Issue do
   alias DevWizard.GithubGateway.{User, Comment, Milestone}
 
-  @linked_issue_regex ~r/(closes|connects)\s+thinkthroughmath\/storyboard#([0-9]+)/i
+  @linked_issue_regex ~r/(closes|connect|fixes)(s)?(ed)?( to)?\s+(thinkthroughmath\/storyboard#|(http|https):\/\/github.com\/thinkthroughmath\/storyboard\/issues\/)([0-9]+)/i
```
Comment (srobbibaro) [inline lib/dev_wizard/github_gateway/issue.ex:4]: 👍

**Comment by srobbibaro** [comment]
LGTM here! 🍔

**Comment by seejee** [comment]
I've added the new env var to all the heroku apps, so this should work once merged and deployed.

**Comment by joelmccracken** [comment]
LGTM. Seej you can merge and deploy.

File: lib/dev_wizard/github_gateway/cache.ex (hunk 5 above)
Comment (joelmccracken) [inline lib/dev_wizard/github_gateway/cache.ex:19]: Love it.

**Pull request merged**